libs/sdk-ui-catalog/styles/scss/detail.scss

```
@@ -3,7 +3,7 @@
 @use "@gooddata/sdk-ui-kit/styles/scss/variables" as kit-variables;
 
 @use "./objectType.scss";
-@use "./parameter.scss";
+@use "./codeDialog.scss";
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@use` (scss/load-partial-extension)

(scss/load-partial-extension)